### PR TITLE
Routes and Navbar

### PR DESCRIPTION
**Pull request by ParthParolekar**
PR for reviewing the code for Navbar and Routes

Also, check if all the links are working and routes are visible

Files to be checked:
- components/Navbar/Navbar.js
- App.js

Live: https://deploy-preview-1--electron-ecommerce.netlify.app/

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,22 @@
...
-        </div>
-      </header>
+      <BrowserRouter>
+        <Navbar />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/Authentication" element={<Authentication />} />
+          <Route path="/Browse" element={<Browse />} />
+          <Route path="/Cart" element={<Cart />} />
+          <Route path="/Wishlist" element={<Wishlist />} />
```
Comment (foolhardy21) [inline src/App.js]: Lowercase letters should be used in path as url is always in lowercase.

**Review by foolhardy21 — APPROVED** [review]
Rest is good to go ahead.

File: src/App.js (hunk 1 above)
Comment (ParthParolekar) [reply src/App.js]: Done

File: src/components/Navbar/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        <i
+          onClick={() => setNavLinksVisible(!navLinksVisible)}
+          className="fas fa-bars hamburger"
+          id="hamburger"
+        ></i>
+
+        <Link to={"/"}>
+          <img src={bigLogo} className="big-logo" alt="logo" />
+        </Link>
+        <Link to={"/"}>
```
Comment (partha8) [inline src/components/Navbar/Navbar.js]: I don't think {} is needed, in the <Link> tag. You can simply do this .. to="/" and it will work.

**Review by partha8 — APPROVED** [review]

File: src/components/Navbar/Navbar.js (hunk 2 above)
Comment (ParthParolekar) [reply src/components/Navbar/Navbar.js]: Done

**Comment by ParthParolekar** [comment]
Thank you @partha8 and @foolhardy21 for the reviews. Merging now

**Pull request merged**